storage/postgres/query_builder_test.go

```diff
@@ -245,9 +243,9 @@ LIMIT 10;`)))
 				criteria5 := query.ByLabel(query.InOperator, "left2", "right2", "right3")
 				criteria6 := query.ByLabel(query.NotEqualsOperator, "left3", "right4")
 
-				_, err := qb.NewQuery().
+				_, err := qb.NewQuery(entity).
```

Comment:
The idea of the multiple criteria test is to document how a query with all possible criterias will look like. Add limit and order as well